packages/mongodb-memory-server-core/src/MongoMemoryServer.ts

```diff
@@ -127,16 +134,19 @@ export default class MongoMemoryServer {
    * @private
    */
   async _startUpInstance(): Promise<MongoInstanceDataT> {
-    const data: any = {};
+    /** Shortcut to this.opts.instance */
+    const instOpts = this.opts.instance ?? {};
     let tmpDir: DirResult;
+    const data: any = {
```

Comment:
is this data object really wanted to be `any` instead of an own interface / properly typed?

Reply:
changed in 70fbdabeb1a34153f59567619b0c5c09bb70a8d0, are you ok with the solution or did i miss something?